dotcom-rendering/package.json

```diff
@@ -51,7 +51,7 @@
 		"@guardian/ophan-tracker-js": "2.2.5",
 		"@guardian/shimport": "1.0.2",
 		"@guardian/source": "8.0.0",
-		"@guardian/source-development-kitchen": "11.0.0",
+		"@guardian/source-development-kitchen": "12.0.0",
```

Comment:
the latest version of the Ticker removes padding from the top and bottom

Reply:
Thanks for doing this Tom